src/lib/components/colors/Header.svelte

```diff
@@ -88,7 +96,16 @@
   data-column="tool"
   data-needs-changes={hasError}
 >
-  <div class="swatch {type}"></div>
+  <div
+    role="complementary"
+    class="swatch {type}"
+    draggable="true"
+    ondrop={switchColorsT}
```

Comment:
In practice, this means that colors are switched whenever _anything_ is dropped onto one of the colors. E.g. I can drop an image, URL, etc. Can we limit it to just when the other color is dropped?

Reply:
I updated it to check that it was the other swatch on `dragover` and `ondrop`.